release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -3021,13 +3021,47 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.18.12
+[id="ocp-4-18-12_{context}"]
+=== RHSA-2025:4427 - {product-title} {product-version}.12 bug fix update and security update
+
+Issued: 08 May 2025
+
+{product-title} release {product-version}.12 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHSA-2025:4427[RHSA-2025:4427] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2025:4429[RHBA-2025:4429] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory.
+
+You can view the container images in this release by running the following command:
+
+[source,terminal]
+----
+$ oc adm release info 4.18.12 --pullspecs
+----
+
+[id="ocp-4-18-12-bug-fixes_{context}"]
+==== Bug fixes
+
+* Previously, a race condition in the hostname that handled the code caused inconsistencies between the boot and machine hostnames. With this release, the race condition is resolved, which ensures consistent hostnames in the Ignition configuration file during the operating system installation. (link:https://issues.redhat.com/browse/OCPBUGS-55364[OCPBUGS-55364])
+
+* Previously, a version of the installation program produced boot image update failures and region-specific user issues because of missing Amazon Machine Image (AMI) IDs in certain regions. With this release, when the region AMI is not found, the region defaults to the `us-east-1` AMI, and the installation program has reliable, default AMIs for all regions. (link:https://issues.redhat.com/browse/OCPBUGS-55290[OCPBUGS-55290])
+
+* Previously, when viewing the list of installed Operators, and the currently selected project matches an Operator's default namespace while copied cluster service versions (CSV) are disabled in the Operator Lifecycle Manager (OLM), an Operator appeared twice in the list. With this release, the Operator appears one time. (link:https://issues.redhat.com/browse/OCPBUGS-55195[OCPBUGS-55195])
```

Comment:
> and the currently selected project matches an Operator's default namespace while copied cluster service versions (CSV) are disabled in the Operator Lifecycle Manager (OLM)

I have no idea what that means. Maybe it's just me ;-)

Reply:
@mburke5678 I just checked with the Jira's assignee and other than removing a comma, he verified that it was correct. It sounds a bit gnarly  since there are so many conditions.